.gitignore

```diff
@@ -12,3 +12,4 @@
 *.env
 *.env_*
 !/tests/fixes/fixtures/*.env
+!/benches/fixtures/*.env
```

Comment:
Why these changes is needed?

Reply:
Was a mistake, thanks for reporting this :)